armbian/base/scripts/autosetup-ssd.sh

```diff
@@ -0,0 +1,172 @@
+#!/bin/bash
```

Comment:
When testing this PR, I noticed that the `+x` flag seems to not be set for this script; unlike other scripts in `armbian/base/scripts/`, this script has permissions `644` at time of writing.

I'd recommend making the permissions consistent, and since we invoke it with `/opt/shift/scripts/autosetup-ssd.sh apply auto --force` below, I believe we'll need `755` or similar.

Reply:
Fixed